src/main/java/com/example/java/test/junior/developer/service/AuthService.java

```diff
@@ -25,6 +26,16 @@ public LoginResponseDto generateLoginResponse(String email, String password) {
       log.error("Error occurred while generating login response for email: {}", email, ex);
       throw new ResponseStatusException(HttpStatus.UNAUTHORIZED, "Invalid credentials", ex);
     }
+
   }
+
+  public String getAccessTokenFromHeader(String authorizationHeader) {
+    String token = null;
+    if (StringUtils.hasText(authorizationHeader) && authorizationHeader.startsWith("Bearer ")) {
```

Comment:
I don't understand. You receive "Bearer bla-bla-bla" in controller. Now you extract "bla-bla-bla". BUT in client I have seen `"Bearer " + accessToken` - don't you feel that you are doing some unnecessary steps...?